about.html

```diff
@@ -0,0 +1,99 @@
+                <li><a href="about.html" onclick="removeStuffs()" class="nav-item">About Conference</a></li>
+                <li><a href="about.html/#past-events" onclick="removeStuffs()" class="nav-item">Past Events</a></li>
+            </ul>
+        </nav>
+    </div>
+    <section class="intro container">
+        <i class="fa-solid fa-bars hamburger"></i>
+        <p class="intro-Text">"GDIW 2022"</p>
+        <h1>Glocalizing Ghana's Digital & Innovation Ecosystem: Opportunities for Economic Development</h1>
+        <div class="border-white intro-desc">
+            <p>A trilateral program to celebrate the accomplishments of Ghana's digital innovation ecosystem and build momentum towards a common vision for its future development.</p>
+        </div>
+        <p>Please contact us via email for any further questions about GDIW 2022! <br> <span><a class="email" href="mailto:hi@gdiw.com.gh">hi@gdiw.com.gh</a></span></p>
+    </section>
+
+    <section id="brief" class="container">
+        <h3>Ghana Digital Innovation Week 2022 </h3>
+        <hr class="divider">
+        <p>The Ghana Digital Innovations Week commenced last year, 2021, It was convened by His Excellency the Vice-President of Ghana and other key ecosystem actors and enablers as part of a trilateral program to celebrate the accomplishments of Ghana’s digital innovation ecosystem and build momentum towards a common vision for its future development and was convened.</p>
+        <div class="border-white intro-desc">
+            <img src="img/Nov-GDIW-LOGO.svg" alt="GDIW 2022 Logo" class="gdiwLogo">
+        </div>
+    </section>
+    <section id="past-events" class="container">
+        <h3> See Past GDIW Reports </h3>
+        <hr class="divider">
+        <p class="past-events-desc">Take a look at previosly organised Ghana Digital Innovation Week Conference <br> Since it's Inception in 2020 in Accra, Ghana</p>
+    <div class="conference-overviews">
+        <div class="conference-overview1">
+            <h3>2020</h3>
+            <p>GDIW Summit 2020 in Accra, Ghana</p>
+        </div>
+        <div class="conference-overview2">
+            <h3>2020</h3>
+            <p>GDIW Summit 2020 in Accra, Ghana</p>
+        </div>
+    </div>
+    </section>
+    <section id="partner" class="container">
+        <h3>Partners</h3>
+        <hr class="divider">
+        <div class="logos">
+            <img src="img/Google-Logo.wine.png" alt="Google Logo">
+            <img src="img/Airbnb-Logo.wine.png" alt="Airbnb Logo">
+            <img src="img/Stripe_(company)-Logo.wine.png" alt="Stripe Logo">
+            <img src="img/Facebook-Logo.wine.png" alt="Facebook Logo">
+        </div>
+    </section>
+    <footer class="container">
+        <img src="img/Nov-GDIW-LOGO.svg" alt="GDIW 2022 Logo">
+        <div>
+            <p>THE GDIW 2022 | All Rights Reserved</p>
+        </div>
+    </footer>
```

Comment:
[OPTIONAL]
- [ ] Great job on making the footer's `background-color` dark in the about page, however it's light in the mobile version.  please consider implementing this since we're supposed to follow the design.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/196691677-0b147aca-cae8-4b1d-9250-c4f45701d5cf.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/196691724-9953c816-414f-43f3-8fbc-20d2c84af720.png)